### PR TITLE
Add 'flip' as a contact task to search kit

### PR DESCRIPTION
**Pull request by colemanw**
This action flips first & last names.

Note: This adds it as an action on Contact searches. With a little tweak it could be added to one of the entities in this extension which are contact-like.

Depends on https://github.com/civicrm/civicrm-core/pull/20307

image

### CODE REVIEW

**Comment by colemanw** [comment]
@eileenmcnaughton I rebased this for you.

**Comment by eileenmcnaughton** [comment]
thanks @colemanw - I started digging into it all yesterday & the clever join thing works a treat

File: ang/dedupeSearchTasks.ang.php [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?php
+// This file declares an Angular module which can be autoloaded
+// in CiviCRM. See also:
+// http://wiki.civicrm.org/confluence/display/CRMDOC/hook_civicrm_angularModules
```
Comment (eileenmcnaughton) [inline ang/dedupeSearchTasks.ang.php:4]: erm - maybe not - where is that boilerplate coming from?

**Comment by eileenmcnaughton** [comment]
I tested this & without civicrm/civicrm-core#20307 the new action is not visible and all & with that pr it works. I was worried that it might offer something that didn't work if I merged it in advance of that being deployed but it seems not so merging - thanks !!!

File: ang/dedupeSearchTasks.ang.php (hunk 1 above)
Comment (colemanw) [reply ang/dedupeSearchTasks.ang.php:4]: Civix

Comment (eileenmcnaughton) [reply ang/dedupeSearchTasks.ang.php:4]: @colemanw old version?

**Pull request merged**